frontend/src/scenes/toolbar-launch/AuthorizedUrls.scss

```diff
@@ -0,0 +1,39 @@
+.AuthorizedUrlRow {
+    display: flex;
+    align-items: center;
+    margin-bottom: 0.5em;
+
+    .LemonTag {
+        margin-right: 1em;
+    }
+    &.suggestion {
+        color: var(--text-muted-alt);
+    }
+    .Url {
+        flex-grow: 1;
+
+        span {
+            max-width: 80%;
+        }
+    }
+    .Actions {
+        display: flex;
+        flex-direction: row;
+
+        .ActionButton {
+            margin-right: 1.5em;
+        }
+
+        & > :last-child {
+            margin-right: 0.75em;
+        }
+    }
+}
+
+.AuthorizedURLForm {
+    width: 100%;
+
+    .form-submit {
+        margin-left: 8px;
+    }
+}
```

Comment:
> Key Results: 50% reduction in custom and inline CSS styles.

How much of this can you remove by using the helper css classes from `global.scss`, like `mr` or `flex-center`?